src/api/cmd/chart-repo-sync/sync.go

```diff
@@ -100,15 +108,47 @@ func sync(repoName, repoURL string) error {
 		return err
 	}
 
+	// Process 10 charts at a time
+	numWorkers := 10
```

Comment:
do you think making this a configurable parameter be a good idea?

Reply:
I was thinking about it, yeah. I think for now this is OK to hardcode, we can make it configurable if there becomes a need for that later